extension/docs/PaypalIntegration.md

```diff
@@ -0,0 +1,37 @@
+## Paypal payment
+
+<!-- START doctoc generated TOC please keep comment here to allow auto update -->
+<!-- DON'T EDIT THIS SECTION, INSTEAD RE-RUN doctoc TO UPDATE -->
+**Contents**
+
+- [Paypal](#paypal)
+
+<!-- END doctoc generated TOC please keep comment here to allow auto update -->
+
+Adyen documentation: https://docs.adyen.com/developers/payment-methods/paypal
+
+**Please see [Integration Guide](IntegrationGuide.md) first before continuing with this document.**
+
+### Paypal
+
+1. [Enable Paypal in Adyen](https://docs.adyen.com/developers/payment-methods/paypal#prerequisites)
+1. Backend creates a payment with following criteria:
+    * `Payment.paymentMethodInfo.method = 'paypal'`
+    * `Payment.transactions` contains a transaction with `type='Charge' and state='Initial'`
+    * `Payment.custom.fields.returnUrl` contains return URL to which the shopper will be redirected after completion.
+1. Adyen-integration make a payment request and save following information to the payment object:
+    * `Payment.interfaceInteractions` contains request and response with Adyen
+    * `Payment.transactions` with a transaction `type='Charge' and state='Initial'` will be changed to `'Pending'`.
+    * `Payment.custom.fields.redirectUrl`  
+    * `Payment.custom.fields.redirectMethod`
+1. Frontend [redirects shopper](https://docs.adyen.com/developers/payment-methods/paypal#step2redirectshopper) to Paypal.
+1. After shopper finishes the payment, she got redirected back to the shop. Backend gets from the query string a parameter `details.payload` and saves the parameter into the Payment custom field `Payment.custom.fields.payload`
```

Comment:
Could you add CTP update action example?

Reply:
https://github.com/commercetools/commercetools-adyen-integration/pull/40/files#diff-3c72c3a852ffe6f8c6fe79caba926f0bR60